src/content/docs/dropins/cart/tutorials/add-product-lines-to-cart-summary.mdx

```diff
@@ -0,0 +1,136 @@
+              .map((option) => option.label)
+              .join(', ');
+              
+              if(selectedOptions) {
+                const productAttribute = document.createElement('div');
+                productAttribute.innerText = `${attr.code}: ${selectedOptions}`;
+                ctx.appendChild(productAttribute);
+              }
+            } else if (attr.value) {
+              const productAttribute = document.createElement('div');
+              productAttribute.innerText = `${attr.code}: ${attr.value}`;
+              ctx.appendChild(productAttribute);
+            }
+          }
+        })
+      },
+   ```
+
+   This code creates a slot named `ProductAttributes` that displays the custom product attribute `Shipping Notes`, if it is assigned to a product. If the corresponding attribute is found, the slot creates a new `div` element and appends the attribute code and value to the element. The element is then appended to the `ctx` element, which is the product line in the cart summary.
+
+1. Save the file and generate the page to see the changes.
+
+
+</Steps>
+
+</Task>
+
+<Task>
+
+## Display promotional information in the footer of a cart item
+
+Now we'll add information defined in a custom cart price rule to the footer of the `CartSummaryList. container. If the the conditions set in the cart price rules are not met, then no additional information is displayed. For example, if a specific coupon has not been applied or if the subtotal threshold has not been met, then this information is not displayed.
+
+<Diagram caption="Display coupon information">
+   ![Display coupon information](@images/dropins/cart/cart-product-line-footer.png)
+</Diagram>
+
+<Steps>
+
+1. Add a `Header` slot beneath the `ProductAttributes` slot.
+
+   ```javascript
+
+    slots: {
+      ProductAttributes: (ctx) => {
+        ...
+        }
+        
+      Footer: (ctx) => {
+        // Runs on mount
+        const wrapper = document.createElement('div');
+        ctx.appendChild(wrapper);
+
+        // Append Product Promotions on every update
+        ctx.onChange((next) => {
+        wrapper.innerHTML = '';
+
+          next.item?.discount?.label?.forEach((label) => {
+          const discount = document.createElement('div');
+          discount.style.color = '#3d3d3d';
+          discount.innerText = label;
+          wrapper.appendChild(discount);
+        });
+      });
+    },
+   ```
+
+   This code creates a slot named `Footer`, which displays the promotional information defined in the custom cart price rule. If the conditions set in the cart price rule are met, the slot creates a new `div` element and appends the promotional information to the element. The element is then appended to the `ctx` element, which is the product line in the cart summary.
+
+1. Save the file. Add products that total at least $75 and apply the BOO24 coupon code to th cart. The page the rule name beneath each item in the cart.
```

Comment:
```suggestion
1. Save the file. Add products that total at least $75 and apply the BOO24 coupon code to the cart. The page displays the rule name beneath each item in the cart.
```

Reply:
I think you forgot to add a link to the image.

Reply:
It's above the steps.